daemon/api.go

```diff
@@ -664,7 +664,11 @@ func searchStore(c *Command, r *http.Request, user *auth.UserState) Response {
 	}
 
 	theStore := getStore(c)
-	found, err := theStore.Find(&store.Search{
+	// TODO: ultimately we want the common layer that calls the
+	// handlers to create a Background context and this to operate
+	// on that
+	ctx := store.WithClientUserAgent(context.TODO(), r.Header.Get("User-Agent"))
```

Comment:
instead of `context.TODO()` we should be calling `Context()` on the `*http.Request` in all new code